homeassistant/components/lovelace/__init__.py

```diff
@@ -18,10 +18,24 @@
 LOVELACE_CONFIG_FILE = 'ui-lovelace.yaml'
 JSON_TYPE = Union[List, Dict, str]  # pylint: disable=invalid-name
 
+FORMAT_YAML = 'yaml'
+FORMAT_JSON = 'json'
+
 OLD_WS_TYPE_GET_LOVELACE_UI = 'frontend/lovelace_config'
 WS_TYPE_GET_LOVELACE_UI = 'lovelace/config'
+
 WS_TYPE_GET_CARD = 'lovelace/config/card/get'
 WS_TYPE_SET_CARD = 'lovelace/config/card/set'
+WS_TYPE_ADD_CARD = 'lovelace/config/card/add'
+WS_TYPE_MOVE_CARD = 'lovelace/config/card/move'
+WS_TYPE_MOVE_CARD_TO_VIEW = 'lovelace/config/card/view'
+WS_TYPE_DELETE_CARD = 'lovelace/config/card/delete'
+
+WS_TYPE_GET_VIEW = 'lovelace/config/view/get'
+WS_TYPE_SET_VIEW = 'lovelace/config/view/set'
```

Comment:
We should not support this. People can move individual cards but shouldn't be able to do many cards at once.

Reply:
The idea was that you would get the config of the view, without the cards, and could edit the name and icon, etc. But that was for a different PR, so should not have been in here...

Reply:
The idea was that you would get the config of the view, without the cards, and could edit the name and icon, etc. But that was for a different PR, so should not have been in here...

Reply:
Good point. We should make a update command for view, but it should exclude cards then.